### ISSUE TITLE
DPS of characters are displayed wrong since the last update

### ISSUE THREAD

**Issue by Sallazarowns** [PathOfBuildingCommunity/PathOfBuilding#4563]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

My 2 builds showed 65+ million and 41+ million DPS respectively. 

### What is the actual behaviour/value?

After update the values don't go over 11 million

### How to reproduce the issue

1. Import skills, jewels, items and passive tree
2. Go to skills tab
3. Select all skills categories into FULL DPS

### Character build code

```shell
https://pastebin.com/vsS2SFak - cyclone slayer (showed 65 million full DPS)
https://pastebin.com/MBGyGb7n - tornado shot deadeye (showed 41 million full DPS)
```

**Comment by QuickStick123** [comment]
This is because of the new Uber Pinnacle Boss config option which applies 70% less damage taken to enemies.
![image](https://user-images.githubusercontent.com/31533893/178465275-a30a22d7-22af-4a5b-94c7-f24b446f531e.png)
Setting to guardian/pinnacle resolves this issue.
![image](https://user-images.githubusercontent.com/31533893/178465385-27517faa-a3cc-4ef2-99fd-0ae49c0f553b.png)

**Comment by Sallazarowns** [comment]
Haven't noticed that change. Sorry.

Some DPS is still missing, but I guess that's the new way of calculating the damage reduction of the boss? I used to run with Sirus A9 iirc.

**Comment by adracea** [comment]
I'm pretty sure the issue should still be active. I'm seeing incredibly lower numbers on Righteous Fire DoT builds. Drops from 6 mil to under 1 mil. The issues seems to be that some player applied debuffs don't do anything (fire exposure for example don't do anything to fire trap dps).

**Comment by deathbeam** [comment]
You need to switch from uber pinnacle boss to guardian/pinnacle boss in config:

![image](https://user-images.githubusercontent.com/5115805/178493926-b99a136b-f95c-4f5e-9bdd-d1b063784dc8.png)

**Comment by adracea** [comment]
![image](https://user-images.githubusercontent.com/6214663/178496189-be9d783f-54f5-4309-8aaf-03cd5bea3ba4.png)


Side by side comparison of the same build loaded in 3.18.1 vs 3.19.0

**Comment by LocalIdentity** [comment]
Fixed by #4581 and #4574

**Comment by QuickStick123** [comment]
Fixed in most recent release.
